src/components/icon/icon.jsx

```diff
@@ -11,6 +11,13 @@ export default function Icon(props) {
     props.currentColor === 'skip' ? '' : styles[`color-${props.currentColor}`],
   ].filter(Boolean);
 
+  const tabindex = props.role === 'button' ? '0' : undefined;
```

Comment:
I just realized this might have unintended consequences. 

We have to actively manage the page tab sequence in complex components. For example, in a taglist, the taglist component ensures only 1 element within it has `tabindex` of `0` (else it is `-1`). This ensures that a keyboard-only user can navigate through the page quickly without having to go through each single tag in a large taglist.

Looks like I can't verify this in the staging branch -- we need to go through the PR checklist btw. 

If it's odd, we can discuss how to architect this a bit better -- we already have concepts of a component being "active" or "inactive" which is tied to `tabindex` as well.